bazel/foreign_cc/luajit.patch

```diff
@@ -65,11 +43,31 @@ index f56465d..3f4f2fa 100644
 
  ##############################################################################
 EOF
-
-patch -p1 < luajit_make.diff
-
-# Default MACOSX_DEPLOYMENT_TARGET is 10.4, which will fail the build at link time on macOS 10.14:
-# ld: library not found for -lgcc_s.10.4
-# This doesn't affect other platforms
-MACOSX_DEPLOYMENT_TARGET=10.6 DEFAULT_CC=${CC} TARGET_CFLAGS=${CFLAGS} TARGET_LDFLAGS=${CFLAGS} \
-  CFLAGS="" make V=1 PREFIX="$THIRDPARTY_BUILD" install
+diff --git a/build.py b/build.py
+new file mode 100755
+index 0000000..b069ef6
+--- /dev/null
++++ b/build.py
+@@ -0,0 +1,21 @@
++#!/usr/bin/env python
++
++import argparse
++import os
++
++def main():
++    parser = argparse.ArgumentParser()
++    parser.add_argument("--prefix")
++    args = parser.parse_args()
++    os.chdir(os.path.dirname(os.path.realpath(__file__)))
++
++    os.environ["MACOSX_DEPLOYMENT_TARGET"] = "10.6"
++    os.environ["DEFAULT_CC"] = os.environ.get("CC", "")
++    os.environ["TARGET_CFLAGS"] = os.environ.get("CFLAGS", "")
++    os.environ["TARGET_LDFLAGS"] = os.environ.get("CFLAGS", "")
++    os.environ["CFLAGS"] = ""
++    os.environ["LSAN_OPTIONS"] = "exitcode=0"
```

Comment:
This one looks new, it's not in the build recipe. How come?

Reply:
since ASAN flags are now propagated, LuaJIT in-house build tool “buildvm” will fail with LSAN (leak detected), setting this env to make sure it doesn’t fail the build itself. Will add a comment.